docs/en/latest/discovery/consul_kv.md

```diff
@@ -73,6 +76,30 @@ The `keepalive` has two optional values:
 - `true`, default and recommend value, use the long pull way to query consul servers
 - `false`, not recommend, it would use the short pull way to query consul servers, then you can set the `fetch_interval` for fetch interval
 
+#### Dump Data
+
+When we need reload `apisix` online, as the `consul_kv` module maybe loads data from CONSUL slower than load routes from ETCD, and would get the log at the moment before load successfully from consul:
+
+```
+ http_access_phase(): failed to set upstream: no valid upstream node
+```
+
+So, we import the `dump` function for `consul_kv` module. When reload, would load the dump file before from consul; when the registered nodes in consul been updated, would dump the upstream nodes into file automatically.
+
+The `dump` has three optional values now:
+
+- `path`, the dump file save path
+    - support relative path, eg: `logs/consul_kv.dump`
+    - support absolute path, eg: `/tmp/consul_kv.bin`
+    - make sure the dump file's parent path exist
+    - make sure the `apisix` has the dump file's read-write access permission,eg: `chown  www:root conf/upstream.d/`
+- `load_on_init`, default value is `true`
+    - if `true`, load the data from dump file before load data from consul
```

Comment:
Look like we need to prepare a dump file for a fresh installed APISIX? Maybe we need to write down this in the docs?

Reply:
Y